autopush/db.py

```diff
@@ -448,7 +448,6 @@ def store_message(self, notification):
             updateid=notification.update_id
         )
         self.table.put_item(data=item, overwrite=True)
-        return True
```

Comment:
now we should stop checking for this result in route_notification too (and remove the silly "result is False" line)

Reply:
AH! I was looking at the wrong .`save_notification` and wondering what you meant. 

And now I'm disturbed that removing that condition and the returned error didn't cause any test to fail or coverage line to be lost. (sigh)